docs/src/index.md

```diff
@@ -68,20 +68,21 @@ in OpenDSS. Here is an example summing the kW and kvar from loads using the
 
 ```julia
 using OpenDSSDirect
-filename = Pkg.dir("OpenDSSDirect", "examples", "8500-Node", "Master.dss")
+filename = joinpath(dirname(dirname(pathof(OpenDSSDirect))),"examples\\8500-Node\\Master.dss")
```

Comment:
We shouldn't use backslashes here or it might break on other platforms. But `joinpath` accepts multiple args:

```suggestion
filename = joinpath(dirname(dirname(pathof(OpenDSSDirect))), "examples", "8500-Node", "Master.dss")
```

(As a sidenote, unless you're handling command-line arguments, you can use forward slash on Windows too.)

Reply:
I agree with your two reviews.